### DISCUSSION TITLE
Unified recommendations for full-width and half-width symbols and related leading and trailing spaces in translation style

### DISCUSSION THREAD

**Discussion by WanderMax** [xiaoyifang/goldendict-ng#982] [Ideas]
简体中文翻译中发现了有一些全角半角符号（括号、冒号）和与之相关的前后空格的翻译风格不统一的问题。
例如：
XML 文件 (*.xml)——XML文件(*.xml)
朗读词条(Alt+S)——选择一个群组(Alt+G)
基于 Qt %1 (%2, %3 位)——简体转正体（台湾）
anki: 发布成功——Anki 搜索：AnkiConnect 未启用。

诸如上述的翻译风格，建议在时间充裕前提下，逐步统一。

参考文档如下：
Microsoft 文风指南
https://www.microsoft.com/zh-cn/Language/StyleGuides?rtc=1

[a comment by an automated account is not shown]

**Comment by xiaoyifang** [comment]
现在的是什么样子的，实际应该是什么样子的，举几个特定的例子比较好对比。

[a comment by an automated account is not shown]

**Comment by WanderMax** [comment]
Software localization: Use English colon (:) in Software localization. A single byte space 
should be left between the English colon and the subsequent characters (Chinese full-width punctuation marks are excluded

Software localization: Use English parentheses ( ) in Software localization. There is no 
space between the parentheses and the Simplified Chinese characters outside of them. 
A single byte space should be left between the parentheses and the 
English characters/number outside of them.

按照微软的这一份指南，软件翻译里面都是用 半角的括号和冒号。是否加空格取决于前后是否是英文字符或数字

XML 文件 (.xml)/XML文件(.xml)：XML 文件(.xml)
朗读词条 (Alt+S)：朗读词条(Alt+S)
选择一个群组(Alt+G)：不变
基于 Qt %1 (%2, %3 位)：基于 Qt %1 (%2，%3 位)
简体转正体（台湾）：简体转正体(台湾)
anki: 发布成功：anki:发布成功
Anki 搜索：AnkiConnect 未启用。：Anki 搜索: AnkiConnect 未启用

anki 是否需要统一翻译为 Anki？

**Comment by xiaoyifang** [comment]
目前没有这么严格的翻译要求，原则上保持视觉上差别不大就可以了。
当然如果有时间处理，也不反对。

> anki 是否需要统一翻译为 Anki？

可以



上面的翻译类的，最好放到代码```里面，否则可能会被github格式化
